ops/cli.py

```diff
@@ -160,6 +160,21 @@ def run_cloudsql_proxy(project: str, cloudsql_instance_name):
                         f'-token="{token}" -instances="{instance}=tcp:0.0.0.0:5432"'])
     return subprocess.check_output(docker_command, shell=True, encoding='utf-8').strip()
 
+def infer_cloudsql_name(project: str):
+    """Try to find a single cloudsql instance with a truthy 'wfl' tag."""
+    info("=> Finding cloud_sql_proxy instance")
+    gcloud_command = f"gcloud --format=json --project {project} sql instances list"
+    instances = json.loads(subprocess.check_output(gcloud_command, shell=True, encoding='utf-8').strip())
+    instance_names = instances \
+        .filter(lambda i: i.get("settings", {}).get("labels", {}).get("wfl", False)) \
+        .map(lambda i: i["name"])
```

Comment:
This looks like Java-ish dot chain style, and I really doubt if this would work (assuming `instance` is a `Union[List, Dict]` type) 🤔 

In Python, `filter` and `map` takes the `func` as the 1st argument and the collection as the second one, and return lazy generators. If `map(filter(func, coll))` is too ugly here, I'd use a custom generator that composes the 2 operations, or look into `functools` (for something like `partialmethod`)

Reply:
Yeah this doesn't work at all, I really wish IntelliJ didn't autocomplete it for me, feels like I'm fighting against the computer.

Reply:
Okay I tried to use list comprehension to do it, hopefully that works